example/passthrough_hp.cc

```diff
@@ -928,8 +942,22 @@ static void do_read(fuse_req_t req, size_t size, off_t off, fuse_file_info *fi)
         FUSE_BUF_IS_FD | FUSE_BUF_FD_SEEK);
     buf.buf[0].fd = fi->fh;
     buf.buf[0].pos = off;
+    enum fuse_buf_copy_flags copy_flags = FUSE_BUF_COPY_FLAGS;
+
+    if (fs.bypass_rw) {
+        /* special benchmark, to see how fast fuse works - just copy
+         * from a memory buffer
```

Comment:
Have you considered splicing the data from /dev/zero instead? That way you could avoid the buffer copy.